__tests__/submission.test.ts

```diff
@@ -0,0 +1,421 @@
+import { expect } from '@jest/globals'
+import { buildSchema } from 'graphql'
+import { IMockServer, mockServer, addMocksToSchema } from '@graphql-tools/mock'
+import { addResolversToSchema } from '@graphql-tools/schema'
+import { initiatilizeFirebaseInstance } from '../src/firebaseDb'
+import resolvers from '../src/resolvers'
+import { Submission } from '../src/typeDefs/dbSchema'
+import { ErrorCode } from '../src/result'
+import fs from 'fs'
+import path from 'path'
+// THIS IMPORT IS NEEDED WHEN YOU WANT TO CLEAR THE DATA FROM EMULATOR
+// import { initializeTestEnvironment, RulesTestEnvironment } from '@firebase/rules-unit-testing'
+// Please install this with @firebase/rules-unit-testing
+
+const schema = buildSchema(fs.readFileSync(
```

Comment:
you can use `loadSchema()` from `schema.ts` file to load it instead of having to write it here

Reply:
I have tried to implement this, but I encounter a type error.
The `addmockSchema` expects that `const schema` is type of `GraphQLSchema`. But the `loadSchema()` has type of DocumentNode. I'm still trying to find a solution but I doubt that this is possible (correct me if I'm wrong).

This is the error: 

_Type 'DocumentNode' is missing the following properties from type 'GraphQLSchema': description, extensions, astNode, extensionASTNodes, and 21 more._

_Type 'DocumentNode' is not assignable to type 'GraphQLSchema'._